main/mandoc/template.py

```diff
@@ -72,3 +71,12 @@ def _base(self):
     self.build_style = "meta"
 
     return []
+
+
+@subpackage("mandoc-apropos")
+def _apropos(self):
+    self.pkgdesc = f"{pkgdesc} (apropos trigger)"
+    self.triggers = ["/usr/share/man"]
+    self.build_style = "meta"
```

Comment:
this is probably wrong, but to make it not so would have to move the makewhatis/apropos symlinks to this